wgkex/client/app.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+import hashlib
+import re
+
+from netlink import find_stale_wireguard_clients, link_handler, WireGuardClient
+
+from dataclasses import dataclass
+from salt.utils.network import mac2eui64
```

Comment:
Is there any other library that can provide this or a similar function?
I'd rather not pull in salt just for this one function.

Reply:
I think we can also just copy the function, might be even more stable for us.

Reply:
https://pypi.org/project/netifaces/ can do that.